packages/website/pages/index.js

```diff
@@ -133,7 +133,7 @@ retrieveFiles()`
           </p>
         </div>
       </div>
-      <div className="md:px-12 pt-24">
+      <div className="layout-margins pt-24">
```

Comment:
I switched to `md:px-12` this so that on mobile the blue box reaches the edges of the screen to give this more space.

i.e. it was like this:

<img src="https://user-images.githubusercontent.com/152863/126902176-95c57980-9170-4d21-80e6-d0cf047299a6.PNG" width="50%" />

...and I switched to this:

<img src="https://user-images.githubusercontent.com/152863/126911979-2fe71f31-16b8-4842-8784-bf8529f90792.png" width="50%" />

We need a `md:layout-margins`, but it's a custom class so need to add to `global.css`.

Reply:
Done